### PR TITLE
Add home items to cart, display items on shopping cart 

### PR DESCRIPTION
**Pull request by lehone-hp**
@muarachmann code review.
- Displayed the cart items on the shopping-cart icon at the top right hand corner
- I used used onclick() on items on the home page when adding to cart. Please review this approach and if you think it is good I will implement on the other pages and update this PR.

### CODE REVIEW

File: resources/views/layouts/app.blade.php [hunk 1]
```diff
@@ -314,6 +322,104 @@
...
+        if(slug && quantity){
+            $.ajax({
+                url: add_to_cart_route,
+                data: data,
+                type: 'POST',
+                success: function (data) {
+                    console.log(data.msg);
+                    alert(data.msg);
+                    $(".cartCount").html(data.cart_count);
+                    updateCartList();
```
Comment (muarachmann) [inline resources/views/layouts/app.blade.php]: Works perfectly ok!

File: resources/views/layouts/app.blade.php [hunk 2]
```diff
@@ -314,6 +322,104 @@
...
+
+            },
+            error: function (xhr, status, error) {
+                console.log(xhr.responseText);
+                alert(error.msg);
+            }
+        });
+
+        //TODO make a request to the server to get all items on the cart
+        //TODO Update the cart_product_list
```
Comment (muarachmann) [inline resources/views/layouts/app.blade.php]: I think its is okay.. Another way to load the cart in all views is to use ( a variable in laravel that can be shared amongst all views). Check on that or we could still go this way which is stressful IMO. Anyway you choose aside that all is good to go

Comment (lehone-hp) [reply resources/views/layouts/app.blade.php]: I don't understand what you mean by using a variable that can be shared in all the views

**Comment by lehone-hp** [comment]
I have updated the PR
- Added a Country model with migration to insert all countries in the table.
- Fill checkout form with user's info i.e. if the user is logged in but leave the fields empty otherwise. 
Review so we can move ahead.

**Comment by muarachmann** [comment]
ok @lehone-hp will test

**Comment by muarachmann** [comment]
Tested and it works fine. Great Job. Just left the remove cart item and condition of if in stock or not. When done i will make a PR for a better toast that alert box dialog

**Comment by muarachmann** [comment]
@lehone-hp you can go ahead and merge this 👍

**Pull request merged**